README.md

```diff
@@ -6,27 +6,23 @@ Collection of utility methods for Android and WordPress.
 
 * In your build.gradle:
 ```groovy
-dependencies {
-    compile 'org.wordpress:utils:1.30.0' // use version 1.30.0
+repositories {
+    maven { url "https://a8c-libs.s3.amazonaws.com/android" }
 }
-```
-
-## Publish an updated version to your local maven repository
-
-You can bump the [version name in the main build file: `WordPressUtils/build.gradle`][1]. After updating the build file, you can build, and publish the library to your local maven repo. That will let you try the new version in your app for example.
 
-```shell
-$ ./gradlew assemble test publishToMavenLocal
+dependencies {
+    implementation 'org.wordpress:utils:1.31.0'
+}
 ```
 
-## Publish it to Bintray
+## Publishing a new version
 
-When a new version is ready to be published to the remote repository, use the following command to publish it to Bintray:
+In the following cases, CircleCI will publish a new version with the following format to our remote maven repo at S3:
 
-```shell
-$ ./gradlew clean build
-$ ./gradlew bintrayUpload -PbintrayUser=FIXME -PbintrayKey=FIXME
-```
+* For each commit in an open PR: `<PR-number>-<commit full SHA1>`
+* Each time a PR is merged to `develop`: `develop-<commit full SHA1>`
+* Each time a PR is merged to `trunk`: `trunk-<commit full SHA1>`
+* Each time a new tag is created: `{tag-name}`
```

Comment:
@mokagio As far as I can tell, this is not yet implemented, is that correct? If it's not, I think we should implement it. What do you think?

Reply:
Correct, it is not. And agreed, there's little change people would get confused by this because of how few external users this library has, but better to be precise and only update the `README` once the code changed, too.

I can work on it if you want, although that will have to wait till the 6th or 7th of April because of Western Easter AFK days.

Reply:
@mokagio This PR is targeting the `publish-to-s3` branch, so I don't think we need to block the merge. I'll take care of the tagging issue before we merge `publish-to-s3` to `develop`.

I've addressed your suggestions, could you take another look?

Reply:
@mokagio It's been 21 days since the last comment, could you please follow up on this?

Reply:
@oguzkocer! OMG I'm so sorry I let this one slip 😢 🙇‍♂️ 🙏

Reply:
No worries :)